packages/components/src/Select/Select.stories.mdx

```diff
@@ -0,0 +1,107 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Select, Option } from "@jobber/components/Select";
+
+<Meta title="Components/Select" component={Select} />
+
+# Select
+
+In Jobber a `Select` is used to present a defined list of options for the user
+to choose from.
+
+```ts
+import { Select, Option } from "@jobber/components/Select";
+```
+
+<Canvas>
```

Comment:
See `Switch` & `Text` suggestions.  Right now these Stories are showing `() =>` before each component so updating them similar to `Switch` will remove that and allow the Canvas section to work
![Screenshot 2023-02-22 at 2 08 37 PM](https://user-images.githubusercontent.com/104797704/220733733-0dee0d8c-7000-40e4-9219-71ccb24af017.png)